README.md

```diff
@@ -233,6 +233,7 @@ Property | Description | Default
 key_id|Internal ID of the KMS encryption key. Can be found via GET to `https://api.instaclustr.com/provisioning/v1/encryption-keys`|""
 alias|KMS key alias, a human-readibly identifier specified alongside your KMS ARN|""
 arn|KMS ARN, identifier specifying provider, location and key in a ':' value seperated string|""
+key_provider|value specifying the the provider account, same as the cluster_provider|INSTACLUSTR
```

Comment:
The wording is a bit strange to me, if it's the same as the cluster_provider then why do we need key_provider?

Reply:
Yea - i agree i don't like the wording - so more than happy to modify just not sure _to what_
They are different scopes is why they are different. 
You're adding a ebs encryption key which relates to a provider - not to a cluster. 
With the cluster_provider stuff, your referencing (maybe) the same provider, but it only relates to that specific cluster.